src/client/src/routes/StyleguideRoute/StyleguideRoute.tsx

```diff
@@ -0,0 +1,73 @@
+import _ from 'lodash'
+import React from 'react'
+import { BrowserRouter, Route, Switch } from 'react-router-dom'
+import { styled, ThemeState } from 'rt-theme'
+
+import FloatingTools from './components/FloatingsTools'
+import { Block, SectionBlock } from './styled'
+
+import Atoms from './sections/Atoms'
+import ColorSpectrum from './sections/ColorSpectrum'
+import CoreBranding from './sections/CoreBranding'
+import FontFamilies from './sections/FontFamilies'
+import Introduction from './sections/Introduction'
+
+const sections = _.mapKeys(
+  {
+    Introduction,
+    ColorSpectrum,
+    CoreBranding,
+    FontFamilies,
+    Atoms,
+    Ending: () => <SectionBlock mh={5} intent="inverted" />,
+  },
+  (value, key) => _.kebabCase(key),
+)
+
+export class StyleguideRoute extends React.Component {
```

Comment:
If you wanted, this looks like it could become a functional component.

Reply:
👍